tests/pytests/integration/states/test_file.py

```diff
@@ -480,6 +480,55 @@ def test_issue_60203(
         )
 
 
+def test_clean_with_wildcard_requisites(
+    salt_master,
+    salt_call_cli,
+    pillar_tree,
+    tmp_path,
+    salt_minion,
+):
+    create_file = tmp_path / "foo"
+
+    sls_name = "file-directory-clean-wildcard"
+    sls_contents = """
+    {tmp_path}:
+      file.directory:
+        - clean: true
+        - require:
+          - file: {tmp_path}/*
+
+    {create_file}:
+      file.managed:
+        - contents: foo bar baz
+    """.format(
+            tmp_path=tmp_path,
+            create_file=create_file,
```

Comment:
```suggestion
        tmp_path=tmp_path,
        create_file=create_file,
```

Reply:
The formatting here is what `black` provided. Should I deviate from that? It'll only get reverted

Reply:
I would stick to black or else the tests will fail. thanks

Reply:
It looks as though our pre-commit check is running black and failing because of these two lines